fips-tests/src/test/java/io/confluent/rest/SslFactoryFipsTest.java

```diff
@@ -72,23 +91,144 @@ public void setUp() {
     }
   }
 
-  @Override
   protected void setConfigs(Map<String, String> configs) {
     configs.put(RestConfig.SSL_PROVIDER_CONFIG, SslFactoryPemHelper.FIPS_SSL_PROVIDER);
     config = new RestConfig(RestConfig.baseConfigDef(), configs);
   }
 
-  @Override
   protected String getKeyStoreType() {
     return SslFactoryPemHelper.FIPS_KEYSTORE_TYPE;
   }
 
-  @Override
   protected String getEncryptedKey() {
     return KEY;
   }
 
   protected Password getKeyPassword() {
     return null;
   }
+
+  @Test
```

Comment:
@trnguyencflt 
My understanding is that both`SslFactoryTest.java` and `SslFactoryFipsTest.java` need to be kept in sync in terms of test-cases, is that correct? If true, can you add this as a comment in each.

Reply:
yup 👍 however, the PR is already merged, I'll add comment in the next PR 👍

Reply:
works, thanks!